src/RestrictedPython/transformer.py

```diff
@@ -1309,37 +1179,45 @@ def visit_While(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        self.not_allowed(node)
 
     def visit_Break(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        return self.node_contents_visit(node)
 
     def visit_Continue(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        self.not_allowed(node)
 
-#    def visit_Try(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+    def visit_Try(self, node):
+        """Allow Try without restrictions.
 
-#    def visit_TryFinally(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+        This is Python 3 only, Python 2 uses TryExcept.
 
-#    def visit_TryExcept(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+        XXX This was forbidden in RestrictedPython 3.x maybe we have to revisit
```

Comment:
Could you please explain this comment ?
In RestrictedPython 3.x the following exception handling constructs
* `try .. except`
* `try .. except .. finally`
* `try ... finally`
are *allowed*.
I have a Zope2.13.23 with RestrictedPython 3.6 in front of me and the following code runs as expected in RestrictedPython
```
errors = ''

try:
    1/0
except Exception as error:
    errors += str(error)
finally:
    errors += 'always runs'
    
try:
    1/0
finally:
  errors += '  another'
  return errors

return 'should not be reached'
```
I know the code does not something useful apart from using exception handling.